opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -135,53 +135,31 @@ pub trait LogProcessor: Send + Sync + Debug {
 /// ```
 #[derive(Debug)]
 pub struct SimpleLogProcessor<T: LogExporter> {
-    exporter: Mutex<T>,
-    is_shutdown: AtomicBool,
+    exporter: T,
 }
 
 impl<T: LogExporter> SimpleLogProcessor<T> {
     pub(crate) fn new(exporter: T) -> Self {
         SimpleLogProcessor {
-            exporter: Mutex::new(exporter),
-            is_shutdown: AtomicBool::new(false),
+            exporter: exporter
```

Comment:
I believe the purpose of Mutex lock was to ensure that the multiple exports are not invoked concurrently.

Reply:
good point! We need to enforce that spec requirement.
Will see how to do it without Mutex.